samples/mobile_run_test_app/Androidruntests/app/src/main/cpp/native-lib.cpp

```diff
@@ -0,0 +1,78 @@
+#include <jni.h>
+#include <android/log.h>
+#include <string>
+#include <unistd.h>
+#include <pthread.h>
+#include <multy_test/run_tests.h>
+#include <multy_core/common.h>
+#include "multy_core/src/utility.h"
+
+//extern "C"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+static int pfd[2];
+static pthread_t thr;
+static const char *tag = "myapp";
```

Comment:
replace that with application name or package name, like "Io.multy.android_run_tests"